src/ripple/app/ledger/Ledger.cpp

```diff
@@ -153,34 +158,37 @@ Ledger::Ledger (Ledger const& ledger,
                 bool isMutable)
     : mParentHash (ledger.mParentHash)
     , mTotCoins (ledger.mTotCoins)
-    , seq_ (ledger.seq_)
-    , mCloseTime (ledger.mCloseTime)
-    , mParentCloseTime (ledger.mParentCloseTime)
     , mCloseResolution (ledger.mCloseResolution)
     , mCloseFlags (ledger.mCloseFlags)
-    , mClosed (ledger.mClosed)
     , mValidated (ledger.mValidated)
     , mAccepted (ledger.mAccepted)
     , mImmutable (!isMutable)
     , txMap_ (ledger.txMap_->snapShot (isMutable))
     , stateMap_ (ledger.stateMap_->snapShot (isMutable))
+    // VFALCO Needs audit
+    , fees_(getFees(*this, getConfig()))
+    , info_ (ledger.info_)
 {
     updateHash ();
 }
 
-// Create a new ledger that follows this one
+// Create a new open ledger that follows this one
 Ledger::Ledger (bool /* dummy */,
```

Comment:
Not in this change set, but it looks like the first parameter here is used like a type-tag parameter. I'd prefer a real tag-type over a bool. (No change needed).